homeassistant/data_entry_flow.py

```diff
@@ -336,6 +336,7 @@ def _async_remove_flow_progress(self, flow_id: str) -> None:
         self._handler_progress_index[handler].remove(flow.flow_id)
         if not self._handler_progress_index[handler]:
             del self._handler_progress_index[handler]
+        flow.async_remove()
```

Comment:
Wrap this in a `try` in case the integration runs into an error.

Reply:
What do we do with the error? Log it?

Reply:
Yes.

Reply:
And include the name of the integration. `Error removing %s config flow: %s`